trace/export.go

```diff
@@ -32,19 +32,20 @@ type Exporter interface {
 
 var (
 	exportersMu sync.Mutex
-	exporters   map[Exporter]struct{}
+	exporters   map[Exporter][]Attribute
 )
 
 // RegisterExporter adds to the list of Exporters that will receive sampled
-// trace spans.
+// trace spans. Passed attributes will be appended to every span that is
+// exported by this exporter.
 //
 // Binaries can register exporters, libraries shouldn't register exporters.
-func RegisterExporter(e Exporter) {
+func RegisterExporter(e Exporter, a ...Attribute) {
```

Comment:
How about using the functional options pattern here instead? So it becomes something like:

`RegisterExporter(e, trace.WithDefaultAttributes(...))`

This keeps our options open w.r.t. adding more arguments in future.